docker-compose-wsl.yml

```diff
@@ -0,0 +1,75 @@
+version: "3"
+services:
+  prisoner-offender-search:
```

Comment:
what's the difference between this file and the `docker-compose.yml`?  When running docker compose one can call multiple files, so would be better just to add the `localstack` specific configuration here and call both using
```
docker-compose -f docker-compose.yml -f docker-compose.wsl.yml up -d
```
or similar.  See https://docs.docker.com/compose/extends/ for more information

Reply:
> "..._the difference between this file and the `docker-compose.yml`?_..."

The Windows one defines [_the `volume` advised by the gist you shared with me the other day_](https://gist.github.com/robfe/9a858b59f4d394ef5deb2517833e75c6#docker-bind-mounts-dont-work-as-well-as-docker-volumes-especially-on-windows).


> "..._When running docker compose one can call multiple files_..."

Thanks Peter. I wasn't aware of that. Good to know. Also, separate files per OS, struck me as simpler (_and quicker_) to get working.

Wouldn't the `bind volume` in one and the `mount volume` in the other, clash somehow?

Reply:
Of course it is simpler and quicker for you to copy all the existing file to get it working, however it ends up being a maintenance nightmare since all changes have to now be made in multiple files.  What normally ends up then happening is that one drifts away from the other one and will eventually stop working altogether.
Can't see why the `bind mount` and `volume mount` will clash since they define the same mount point so that the one defined last will override the other one.  Obviously the only way to be certain is to try it out.